### DISCUSSION TITLE
Interactive markers support

### DISCUSSION THREAD

**Discussion by matthew-t-watson** [foxglove/community#817] [Q&A]
A lack of interactive markers support is the single missing feature preventing us from moving our web UI to Foxglove - is there a plan for adding support for these messages?

**Comment by amacneil** [comment]
Nothing currently planned.

Are you looking for a straight clone of the RViz feature, or would you be able to describe your requirements a little more?

**Comment by matthew-t-watson** [comment]
We only require the rviz 'menu' feature so that we can run commands on markers via the UI. We don't require basic_controls.

**Comment by leeaidan** [comment]
This a feature that we are looking for also! The menu feature in RViz is very useful for sending custom commands to robots without having to publish a message or through other methods.
